fbw-a380x/src/base/flybywire-aircraft-a380-842/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A380/EFB/mapConfig.json

```diff
@@ -0,0 +1,118 @@
+﻿{
```

Comment:
is mapConfig.json a new configuration? Document this in the PR if so

Reply:
it is a copy of the A32NX - no changes but the A380X should have its own copy in its base files